test/test_remote_io.py

```diff
@@ -1,4 +1,3 @@
-# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
why is it removed here?

Reply:
Thanks for pointing it out! I have no idea :P